src/middleware/pnid.ts

```diff
@@ -14,16 +14,17 @@ async function PNIDMiddleware(request: express.Request, response: express.Respon
 	const parts = authHeader.split(' ');
 	const type = parts[0];
 	let token = parts[1];
-	let pnid: HydratedPNIDDocument | null;
+	let pnid: HydratedPNIDDocument | null = null;
 
 	if (request.isCemu) {
 		token = Buffer.from(token, 'hex').toString('base64');
 	}
 
-	if (type === 'Basic') {
+	if (type === 'Basic' && request.path.includes('v1/api/people/@me/devices')) {
 		pnid = await getPNIDByBasicAuth(token);
-	} else {
-		pnid = await getPNIDByTokenAuth(token);
+	} else if (type === 'Bearer') {
+		// TODO - This "accepted types list" is mostly a hack. Change this

```

Comment:
Looks like this comment is old? Seems like it can be removed as this method doesn't have an accepted types list

Reply:
You're right, this was a left-over from an older idea I had